src/cookbook/games/achievements-leaderboard.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Add achievements and leaderboards to your mobile game
+description: >
+  How to use the games_services plugin to add functionality to your game.
+---
+
+Gamers have various motivations for playing games. In broad strokes,
+there are four major motivations: 
+[immersion, achievement, cooperation, and competition](https://meditations.metavert.io/p/game-player-motivations).
+No matter the game you build, some players want to *achieve* in it. This
+could be trophies won or secrets unlocked. Some players want to
+*compete* in it. This could be hitting high scores or accomplishing
+speedruns. These two ideas map to the concepts of *achievements* and
+*leaderboards*.
+
+![A simple graphic representing the four types of motivation explained above](/assets/images/docs/cookbook/types-of-gamer-motivations.png)
+
+Ecosystems such as the App Store and Google Play provide centralized
+services for achievements and leaderboards. Players have achievements
+from all the games in one place and developers don't need to
+re-implement them for every game.
```

Comment:
When reading, I wasn't sure what "all the games" meant. Would something like "players **can view** achievements from **all their** games..." make sense?

```suggestion
Ecosystems such as the App Store and Google Play provide
centralized services for achievements and leaderboards.
Players can view achievements from all their games in one place and
developers don't need to re-implement them for every game.
```